service/collector/helper.go

```diff
@@ -46,11 +55,36 @@ func newHelper(config helperConfig) (*helper, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.AWSConfig must not be empty", config)
 	}
 
+	var clusterStore = cache.NewStore(cache.DeletionHandlingMetaNamespaceKeyFunc)
+	var configStore = cache.NewStore(cache.DeletionHandlingMetaNamespaceKeyFunc)
+	{
+		listerWatcher := &ClusterListerWatcher{
+			clusters: config.G8sClient.InfrastructureV1alpha2().AWSClusters(metav1.NamespaceAll),
```

Comment:
As per https://github.com/giantswarm/giantswarm/issues/9368 we decided to start using the generic controller client more. I would like to use it here as well if that works.

Reply:
We can probably also get rid of those `*ListWatcher` types then.